AirbnbSwiftFormatTool/AirbnbSwiftFormatTool.swift

```diff
@@ -28,6 +28,9 @@ struct AirbnbSwiftFormatTool: ParsableCommand {
   @Flag(help: "When true, source files are not reformatted")
   var lint = false
 
+  @Flag(help: "When true, logs the commands that are executed")
+  var log = false
```

Comment:
This was useful for diagnosing some weird SwiftLint failures I was hitting (some specific files in the Epoxy repo cause it to fail with `Bus error: 10` or `Illegal instruction: 4` for some reason).